src/tests/ftest/util/telemetry_utils.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from logging import getLogger
+from ClusterShell.NodeSet import NodeSet
+
+
+class TelemetryUtils():
+    """Defines a object used to verify telemetry information."""
+
+    def __init__(self, dmg, servers):
+        """Create a TelemetryUtils object.
+
+        Args:
+            dmg (DmgCommand): [description]
+            servers (list): [description]
+        """
+        self.log = getLogger(__name__)
+        self.dmg = dmg
+        self.hosts = NodeSet.fromlist(servers)
+
+    def get_metrics(self, name):
+        """Obtain the specified metric informaton for each host.
+
+        Args:
+            name (str): Comma-separated list of metric names to query.
+
+        Returns:
+            dict: a dictionary of host keys linked to metric data for each
+                metric name specified
+
+        """
+        info = {}
+        self.log.info("Querying telemetry metric %s from %s", name, self.hosts)
+        for host in self.hosts:
+            data = self.dmg.telemetry_metrics_query(host=host, metrics=name)
+            info[host] = {}
+            if "response" in data:
+                if "metric_sets" in data["response"]:
+                    for entry in data["response"]["metric_sets"]:
+                        info[host][entry["name"]] = {
+                            "description": entry["description"],
+                            "metrics": entry["metrics"]
+                        }
+        self.log.debug("INFO = %s", info)
+        return info
+
+    def check_container_metrics(self, open=None, active=None, close=None,
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'open'

Reply:
Fixed